.devcontainer/pyproject.toml

```diff
@@ -5,16 +5,16 @@ description = "Epiphany Dev Container"
 authors = ["Epiphany Platform"]
 
 [tool.poetry.dependencies]
-python = "3.10.4"
+python = "3.10.6"
```

Comment:
We define minor version here so how about doing the same (for consistency) in Dockerfile? I mean `FROM python:3.10-slim` -> `FROM python:3.10.6-slim` or using e.g. `python = "~3.10"` here without changing Dockerfile.

Reply:
Yeah that is actually a good plan to also lock down the Python version.